miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/TabbedListItemsPage.java

```diff
@@ -0,0 +1,103 @@
+package uk.ac.bbsrc.tgac.miso.webapp.util;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Project;
+import uk.ac.bbsrc.tgac.miso.core.data.type.PlatformType;
+import uk.ac.bbsrc.tgac.miso.core.manager.RequestManager;
+
+public class TabbedListItemsPage {
+  private static final Logger log = LoggerFactory.getLogger(TabbedListItemsPage.class);
+
+  public static TabbedListItemsPage createForPlatformType(String targetType, String property, RequestManager requestManager)
```

Comment:
If we can get [this reviewed](https://doodle.com/poll/w3kxspsefv9k743z), you'll have a PlatformService and won't need the RequestManager. Or the PlatformService could even be extracted out of that PR.